gas/config/tc-riscv.c

```diff
@@ -1357,6 +1357,13 @@ validate_riscv_insn (const struct riscv_opcode *opc, int length)
 	case '>': USE_BITS (OP_MASK_SHAMT, OP_SH_SHAMT); break;
 	case 'A': break; /* Macro operand, must be symbol.  */
 	case 'B': break; /* Macro operand, must be symbol or constant.  */
+	case 'b': /* CORE-V Specific.  */
```

Comment:
Hmm it would be nice if we could group them together, however I can't find any examples in the file and yeah `x` is unfortunately taken... I guess I can make it clearer with the comment?

Reply:
Does something like this make sense:

```c
	case 'x': /* Vendor-specific operands.  */
	  switch (*++oparg) 
	    {
	      /* Vendor-specific (CORE-V) Xcvmac operands.  */
	      case 'c':
		if (oparg[1] == 'b')
		  {
		    used_bits |= ENCODE_CV_L_UIMM5(-1U);
		    ++oparg; break;
		  }
		break;
	      default:
		goto unknown_validate_operand;
	    }
	  break;
```

This translates to `xcb` though, not `xcvb`. Would you suggest I just iterate again or is there a better way?

Reply:
`xcb` might be ok... given 95 printable ascii characters that allows for 95 vendors under `x`. You could go with this and it could be discussed further upstream if needed.

`'b` will be used for other CORE-V extensions, so you could remove `Xcvmac` from the comment.

Reply:
I've tried implementing it in this commit (c09d93ed01e88fd1b0a8d24e613792189f468795), however the `cv-mac-insns` tests are failing due to "undefined modifier (x)". The exact error message is in the commit. I'll investigate but you might find something obviously wrong before me :)

Reply:
I successfully implemented this [here](1233f84ac5b38f2e63758e515165974d47872396). I have run `make check-gas` and can confirm everything passes. I'm happy to squash, thought leaving it separate for now would be better for reviewing though.

Reply:
Now that I got to implementing this in ALU, I'm thinking: is `xcb` the best name for the Xcvmac operand? If so, what should I name the Xcvalu operand? They were `b1` and `b3` respectively. @MaryBennett what do you think about this?

Reply:
This thread is resolved except for the naming discussion.